tidb-cloud/migrate-from-mysql-using-data-migration.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Migrate MySQL-Compatible Databases to TiDB Cloud Using Data Migration
+summary: Learn how to migrate data from MySQL-compatible databases hosted in Amazon Aurora MySQL, Amazon Relational Database Service (RDS), or a local MySQL instance to TiDB Cloud using Data Migration.
+---
+
+# Migrate MySQL-Compatible Databases to TiDB Cloud Using Data Migration
+
+This document describes how to migrate data from a MySQL-compatible database on a cloud provider (Amazon Aurora MySQL or Amazon Relational Database Service (RDS)) or on-premises to TiDB Cloud using the Data Migration feature of the TiDB Cloud console.
+
+This feature allows you to migrate your database and its ongoing changes to TiDB Cloud (either in the same region or cross regions) with minimal downtime
+
+## Limitations
+
+- Currently, the Data Migration feature is in beta, and you can create only one migration job for free for each organization. To create more migration jobs, you need to [file a support ticket](/tidb-cloud/tidb-cloud-support.md).
+
+- The Data Migration feature is only available to clusters created in the AWS `Oregon (us-west-2)` and AWS `Singapore (ap-southeast-1)` regions after November 9, 2022. If your cluster was created before the date or if your cluster is in another region, this feature is not available to your cluster and the **Data Migration** tab will not be displayed on the cluster overview page in the TiDB Cloud console.
+
+- The Data Migration feature is available only for Dedicated Tier clusters.
+
+- The system databases will be filtered out and not migrated to TiDB Cloud even if you select all of the databases to migrate. That is, `mysql`, `information_schema`, `information_schema`, and `sys` will not be migrated using this feature.
+
+- If the table to be migrated already exists in the target database, TiDB Cloud appends the data to the target table directly. If the keys conflict, an error is reported.
+
+- When you delete a cluster in TiDB Cloud, all migration jobs in that cluster are automatically deleted and not recoverable.
+
+- During incremental replication (migrating ongoing changes to your cluster), if the migration job recovers from an error, it will enable a safe mode. In this mode, the migration job applies the binlogs which are up to 60 seconds before the breakpoint to the target database. In this mode, `INSERT` statements are replicated as `REPLACE`, `UPDATE` statements as `DELETE` and `REPLACE`, and then these transactions are replicated to the downstream cluster to make sure all the data during the breakpoint has been migrated to the downstream cluster. When the table does not have primary keys or not-null unique indexes, some data might be duplicated because the data is inserted repeatedly.
```

Comment:
```suggestion
- During incremental replication (migrating ongoing changes to your cluster), if the migration job recovers from an abrupt error, it may open the safe mode for 60 seconds. During safe mode, `INSERT` statements are replicated as `REPLACE`, `UPDATE` statements as `DELETE` and `REPLACE`, and then these transactions are replicated to the downstream cluster to make sure all the data around the abrupt error has been migrated smoothly to the downstream cluster. When the table does not have primary keys or not-null unique indexes, some data might be duplicated because the data is inserted repeatedly.
```

Reply:
错误重启，safe mode 并不是一定会进入的，取决于 DM 是否正常退出。